README.md

```diff
@@ -1,55 +1,60 @@
+# <a name="title"></a> Kitchen::OpenStack: A Test Kitchen Driver for OpenStack
+
 [![Gem Version](https://img.shields.io/gem/v/kitchen-openstack.svg)][gem]
 [![Build Status](https://img.shields.io/travis/test-kitchen/kitchen-openstack.svg)][travis]
 [![Code Climate](https://img.shields.io/codeclimate/github/test-kitchen/kitchen-openstack.svg)][codeclimate]
 [![Coverage Status](https://img.shields.io/coveralls/test-kitchen/kitchen-openstack.svg)][coveralls]
 [![Dependency Status](https://img.shields.io/gemnasium/test-kitchen/kitchen-openstack.svg)][gemnasium]
 
-[gem]: https://rubygems.org/gems/kitchen-openstack
-[travis]: https://travis-ci.org/test-kitchen/kitchen-openstack
-[codeclimate]: https://codeclimate.com/github/test-kitchen/kitchen-openstack
-[coveralls]: https://coveralls.io/r/test-kitchen/kitchen-openstack
-[gemnasium]: https://gemnasium.com/test-kitchen/kitchen-openstack
+A [Test Kitchen][kitchenci] Driver for [OpenStack][openstack_web].
```

Comment:
you've got some markdown errors here, I'm assuming you want these to be links somewhere.